tests/functional/api/conftest.py

```diff
@@ -0,0 +1,66 @@
+import base64
```

Comment:
These are heavily based off the fixtures extracted from the group tests in https://github.com/hypothesis/h/pull/5957.

This means:

 * They aren't specific to this and represent a decent "average" of the fixtures repeated throughout these API tests
 * These are probably good enough to use in the other tests too when we choose to (with appropriate modification of those tests)
 * These will be re-used when we merge: https://github.com/hypothesis/h/pull/5957